deploy/chart/values.yaml

```diff
@@ -37,6 +37,13 @@ defaultStorageClass:
   enabled: true
 
 deploy:
+  # By default, the controller server does not delete NAS subpaths when DeleteVolume, that means
+  # the files created in the volume will remain in the filesystem after PV deleted.
+  # You can set "enableNasSubpathFinalizer" to true in ACK clusters with storage-operator addon installed, 
+  # then the controller server will patch a finalzier on NAS subpath-type PV when DeleteVolume 
+  # and the storage-operator will handle the cleanup of the subpath in NAS filesystem.
+  enableNasSubpathFinalizer: false
```

Comment:
how about just name it `deploy.withStorageController`? And default it to `deploy.ack`.

Reply:
We should not disable it here, or we will break existing ACK installations.

Reply:
Users would better install csi from ACK addons, where we can set the dependencies on. Using the chart directly with this enabled, users could be confused when their PV stuck in terminating if storage-controller not working as expect.

Reply:
But the manifests we use in ACK addon will be generated from this `values.yaml`. Or should we introduce another values-ack.yaml?